Process panics if "command-working-directory" doesn't exist

Hello

It looks like Webhook panics if the directory set up as `command-working-directory` value doesn't exist:

```
$ ./webhook -verbose -hooks hooks.yaml
[webhook] 2017/12/21 10:24:38 version 2.6.7 starting
[webhook] 2017/12/21 10:24:38 setting up os signal watcher
[webhook] 2017/12/21 10:24:38 attempting to load hooks from hooks.yaml
[webhook] 2017/12/21 10:24:38 found 1 hook(s) in file
[webhook] 2017/12/21 10:24:38   loaded: gitlab-pipeline
[webhook] 2017/12/21 10:24:38 serving hooks on http://0.0.0.0:9000/hooks/{id}
[webhook] 2017/12/21 10:24:38 os signal watcher ready
[webhook] 2017/12/21 10:26:47 [eaae6e] incoming HTTP request from 127.0.0.1:40574
[webhook] 2017/12/21 10:26:47 [eaae6e] gitlab-pipeline got matched
[webhook] 2017/12/21 10:26:47 [eaae6e] gitlab-pipeline hook triggered successfully
[webhook] 2017/12/21 10:26:47 200 | 279.994µs | localhost:9000 | POST /hooks/gitlab-pipeline
[webhook] 2017/12/21 10:26:47 [eaae6e] error creating temp file [open /var/run/webhooks/WEBHOOK_PAYLOAD_FILE596074151: no such file or directory]
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x732da0]

goroutine 24 [running]:
main.handleHook(0xc420114a80, 0xc42012e510, 0x6, 0xc4200f4198, 0xc4200f41a0, 0xc4200f41a8, 0xc4200f13a0, 0xf, 0xc420036501, 0x0, ...)
        /home/adnan/projects/go/src/github.com/adnanh/webhook/webhook.go:372 +0x850
created by main.hookHandler
        /home/adnan/projects/go/src/github.com/adnanh/webhook/webhook.go:296 +0xf71
```

Content of the `hooks.yaml` file (in which there is a typo, `command-working-directory` should have been set to `/var/tmp/webhooks`:

```yaml
- id: gitlab-pipeline
  execute-command: /home/ec2-user/gitlab-pipeline.py
  command-working-directory: /var/run/webhooks
  pass-file-to-command:
  - source: entire-payload
    envname: WEBHOOK_PAYLOAD_FILE
    base64decode: false

  trigger-rule:
    match:
      parameter:
        source: header
        name: X-Gitlab-Event
      type: value
      value: Pipeline Hook
```

--- a/webhook.go
+++ b/webhook.go
@@ -23,7 +23,7 @@ import (
 )
 
 const (
-	version = "2.6.7"
+	version = "2.6.8"
 )
 
 var (
@@ -368,13 +368,16 @@ func handleHook(h *hook.Hook, rid string, headers, query, payload *map[string]in
 		tmpfile, err := ioutil.TempFile(h.CommandWorkingDirectory, files[i].EnvName)
 		if err != nil {
 			log.Printf("[%s] error creating temp file [%s]", rid, err)
+			continue
 		}
 		log.Printf("[%s] writing env %s file %s", rid, files[i].EnvName, tmpfile.Name())
 		if _, err := tmpfile.Write(files[i].Data); err != nil {
 			log.Printf("[%s] error writing file %s [%s]", rid, tmpfile.Name(), err)
+			continue
 		}
 		if err := tmpfile.Close(); err != nil {
 			log.Printf("[%s] error closing file %s [%s]", rid, tmpfile.Name(), err)
+			continue
 		}
 
 		files[i].File = tmpfile

